_episodes/04-reusing_tools.md

```diff
@@ -56,4 +41,88 @@ To implement the last step the bio-cwl-tools library library will be used.
 {: .challenge}
 {% include links.md %}
 
+After finding the `featureCounts` tool, we need to download the tool to the directory in which our workflow is located.
+To download the tool, we go to the `featureCounts.cwl` page and open the raw file. 
+The screenshot below shows where you can find this button.
+
+!{}({{page.root}}/fig/screenshot_featureCounts_tool.png)
```

Comment:
This didn't work for me at https://deploy-preview-55--thirsty-hoover-1e0704.netlify.app/04-reusing_tools/index.html